voting/orbs-pos-data/src/__tests__/staking-service-mock.test.ts

```diff
@@ -24,58 +26,137 @@ function testTxCreatingMethods() {
   });
 }
 
-function testDataReadingMethods() {
-  describe(`Data reading methods`, () => {
-    let stakingService: StakingServiceMock;
+function testDataReadingMethodsWitchRealAPI() {
```

Comment:
rename test that it imitates real functionality

Reply:
Done